source/includes/api/_subscriptions.md

```diff
@@ -1,704 +1,811 @@
+        "createdAt": "2020-01-16T12:45:08.434Z",
+        "userId": "57a063da096c4eda523e99ae",
+        "resource": {
+          "type": "EMAIL",
+          "content": "example@wri.org"
+        },
+        "datasets": ["viirs-active-fires"],
+        "datasetsQuery": [],
+        "params": {
+          "geostore": "50601ff9257df221e808af427cb47701"
+        },
+        "confirmed": false,
+        "language": "en",
+        "env": "production"
+      }
     }
-}'
+  ]
+}
 ```
 
-In this case, a subscription is being created in reference to the `subscribable` field present in the previously defined dataset. After confirming the subscription the `subscriptionQuery` will be ran and its result will be compared against the `threshold`. If the query result is at least equal to the threshold, a new alert will be sent to the subscription's email.
+This endpoint will allow you to get all the subscriptions owned by the user who performed the request (identified by the access token). To use this endpoint, you must be logged in (i.e. a token must be provided). In the sections below, we’ll explore how you can customize this endpoint call to match your needs.
 
-### From country
+For a detailed description of each field, check out the [Subscription reference](#subscription-reference) section.
 
-Field        |             Description             |   Type
------------- | :---------------------------------: | -----:
-params       | Geographic area of the subscription | Object
--- iso       |    Country or region information    | Object
----- country |              Iso code               |   Text
+### Filters
 
-> To create a Subscription, you have to do a POST with the following body:
+The `v1/subscriptions` endpoint supports the following optional query parameters as filters:
 
-```shell
-curl -X POST https://api.resourcewatch.org/v1/subscriptions \
--H "Authorization: Bearer <your-token>" \
--H "Content-Type: application/json"  -d \
- '{
-   "name": "<name>",
-   "application": "<application>",
-   "env": <environment>,
-   "language": "<language>",
-   "resource": {
-       "type": "<type>",
-       "content": "<content>"
-   },
-   "datasets" : ["<dataset>"],
-   "params": {
-       "iso": {
-           "country": "<iso>"
-       }
-   },
-   "datasetsQuery": [{}]
-  }'
-```
+Field       |  Description                                             | Type   | Default value
+----------- | :------------------------------------------------------: | -----: | ---------------:
+application | Application to which the subscription is associated.     | String | 'gfw'
+env         | The environment to which the subscription is associated. | String | 'production'
 
-### From country and region
+**Deprecation notice:** the default value for the `application` filter (currently, `gfw`) will be removed and the `application` filter will then have no default value. We recommend reviewing your application to ensure you set and load the correct `application` explicitly.
```

Comment:
Will be removed when?

Reply:
We actually have no date for removal, though I agree we should plan to remove this. The main intention here is to prevent users from trusting that the `application` filter will always be defaulting to `gfw`.

Subscriptions were built this way from the start, and GFW is currently using it trusting that the application default will be `gfw` (thus, not providing the application param). If we removed this default value, we would also need to do a full-sweep to GFW to ensure that nothing gets broken there from removing this value.